chrome/content/zotero-better-bibtex/schomd.coffee

```diff
@@ -117,8 +191,10 @@ Zotero.BetterBibTeX.schomd.citation = (citekeys, {style, libraryID} = {}) ->
 
   return '' if itemIDs.length == 0
 
-  url = "http://www.zotero.org/styles/#{style ? 'apa'}"
-  style = Zotero.Styles.get(url)
+  urlstyle = "#{style ? 'apa'}"
```

Comment:
Ok, the thing here is that the incoming variable named "style" is being assigned to, and thus the initial value is lost, but it's needed more than one time, to test for a short style name in the http://www.zotero.org/styles/ namespace and if that's not found, to check for one in the http://juris-m.github.io/styles/ namespace, which is where the "jm-babyblue" style is, among others.

The patch to Juris-M for that support has been accepted, I think. Until it's in the released version, passing in the full URL version of the style id works.